app/controllers/recipe_ingredients_controller.rb

```diff
@@ -0,0 +1,6 @@
+class RecipeIngredientsController < ApplicationController
+  def show
+    @recipe = Recipe.find(params[:recipe_id])
+    @ingredients = @recipe.ingredients
```

Comment:
The 3 user stories you completed don't call for any `#show` method of this controller - though you may want to look at Extension # 2 which may use this controller for a different action. 

As it stands, be careful of spending time on things in these ICs that don't directly have to do with the user stories at hand.